lib/app/admin/router.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var rebuildData = require('./data-reset');
+var HttpStatus = require('http-status-codes');
+
+
+function getResetDataHandler(mediator) {
+
+  return function resetData(req, res) {
+    rebuildData(mediator, req.body.remove_list)
+      .then(function() {
+        res.send(HttpStatus.NO_CONTENT);
+      })
+      .catch(function() {
+        res.send(HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Should really respond with an error

`res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(err)`